src/WebJobs.Script/Workers/Http/HttpFunctionInvocationDispatcher.cs

```diff
@@ -65,15 +65,14 @@ internal class HttpFunctionInvocationDispatcher : IFunctionInvocationDispatcher
 
         internal Task InitializeHttpWorkerChannelAsync(int attemptCount, CancellationToken cancellationToken = default)
```

Comment:
Does it mean this fix only works for HttpTrigger? Or will it work on other triggers as well?